src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -398,60 +444,65 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 				htmlGenerator.generate(); // Try to generate docs... hopefully
 				Skript.info(sender, "All documentation generated!");
 			} else if (args[0].equalsIgnoreCase("test") && TestMode.DEV_MODE) {
-				File scriptFile;
+				File[] scriptFiles;
 				if (args.length == 1) {
-					scriptFile = TestMode.lastTestFile;
-					if (scriptFile == null) {
-						Skript.error(sender, "No test script has been run yet!");
+					scriptFiles = TestMode.lastTestFiles;
+					if (scriptFiles == null || scriptFiles.length == 0) {
+						Skript.error(sender, "No test script(s) have been ran yet!");
 						return true;
 					}
 				} else {
 					if (args[1].equalsIgnoreCase("all")) {
-						scriptFile = TestMode.TEST_DIR.toFile();
+						scriptFiles = new File[]{TestMode.TEST_DIR.toFile()};
 					} else {
-						scriptFile = getScriptFromArgs(sender, args, TestMode.TEST_DIR.toFile());
-						TestMode.lastTestFile = scriptFile;
+						scriptFiles = getScriptsFromArgs(sender, args, TestMode.TEST_DIR.toFile());
+						TestMode.lastTestFiles = scriptFiles;
 					}
 				}
 
-				if (scriptFile == null || !scriptFile.exists()) {
-					Skript.error(sender, "Test script doesn't exist!");
+				if (scriptFiles == null || scriptFiles.length == 0)
```

Comment:
why was the exists() check removed? was it unnecessary? Same with the corresponding error message.